evolvedbinary/prosemirror-lwdita
# Feature/branch name

Adds the option of using a different branch as base instead of `main`.

# how to test this PR:
* from a clean state (remove node_modules & any fork) 
* checkout the branch
* install dependencies
* start the demo `yarn start:demo`
* open `http://localhost:1234/?ghrepo=evolvedbinary/prosemirror-lwdita&source=packages/prosemirror-lwdita-demo/example-xdita/02-short-file.xml&branch=ignore/link&referer=htwsrhtshrts`
* the short-example.xml should reflect the current state of the one in the branch `ignore/link`
* Create a new branch on `prosemirror-lwdita`
* change the state of one of the example file
* make a request to `http://localhost:1234/?ghrepo=evolvedbinary/prosemirror-lwdita&source=packages/prosemirror-lwdita-demo/example-xdita/02-short-file.xml&branch=ignore/link&referer=htwsrhtshrts`, replace the `ignore/link` branch with your new branch
* the short-example.xml should reflect the current state of the one in your new branch
* start the backend `yarn start:backend`
* authorize and install petal GitHub app
* make a request to `http://localhost:1234/?ghrepo=evolvedbinary/prosemirror-lwdita&source=packages/prosemirror-lwdita-demo/example-xdita/02-short-file.xml&branch=ignore/link&referer=htwsrhtshrts`
* edit the file 
* click publish
* fill out the form
* check the create pr, it should be created against the base `ignore/link`
